go/rpcclientlib/encrypted_client.go

```diff
@@ -4,19 +4,15 @@ import (
 	"crypto/rand"
 	"encoding/json"
 	"fmt"
-	"strings"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/crypto/ecies"
 )
 
 const (
-	http                = "http://"
-	reqJSONKeyFrom      = "from"
-	reqJSONKeyData      = "data"
-	ethCallPaddedArgLen = 64
-	ethCallAddrPadding  = "000000000000000000000000"
+	http           = "http://"
```

Comment:
This file needs to have a single responsibility, which is the encClient.
All the other stuff should be removed.
It should start with the `EncRPCClient` struct, and the comment that it's a wrapper..

For example these will not be needed after the suggested refactorings.